libkbfs/backpressure_disk_limiter.go

```diff
@@ -785,6 +814,18 @@ func defaultGetFreeBytesAndFiles(path string) (int64, int64, error) {
 	return int64(freeBytes), int64(freeFiles), nil
 }
 
+func (bdl *backpressureDiskLimiter) trackerFromType(typ diskLimitTrackerType) (
```

Comment:
But `journalLimitTracker` is a valid value of the `diskLimitByteTracker` enum, isn't it?